src/css/components/proposal.css

```diff
@@ -0,0 +1,234 @@
+  width: 100%;
+  height: 100%;
+  background-color: var(--lighter-green-color);
+  transform: rotate(-8deg);
+  z-index: -1;
+}
+
+.proposal__content-wrapper::after {
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: var(--white-color-50);
+  transform: rotate(11deg);
+  z-index: -2;
+}
+
+.proposal__content {
+  display: flex;
+  justify-content: space-between;
+  padding: 40px 20px;
+  border-radius: 16px;
+  background-color: var(--white-color);
+}
+
+.proposal__subheader {
+  margin-bottom: 20px;
+  font-weight: 700;
+  font-size: var(--proposal-subheader-font-size-mobile);
+  line-height: 1.2;
+  letter-spacing: -0.02em;
+  color: var(--black-color);
+}
+
+.proposal__feature-list {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+  gap: 16px;
+}
+
+.proposal__feature {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.proposal__icon-bg {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-shrink: 0;
+  width: 28px;
+  height: 28px;
+  border-radius: 50%;
+  background-color: var(--green-color);
+}
+
+.proposal__icon {
+  fill: var(--white-color);
+}
+
+.proposal__text {
+  display: inline-block;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 1.45;
+  letter-spacing: -0.02em;
+  color: var(--black-color);
+}
+
+.proposal_btn {
+  padding: 14px 0;
+  width: 100%;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 1.55;
+  border: none;
+  border-radius: 12px;
+  background-color: var(--green-color);
+  color: var(--white-color);
+  transition: var(--default-trns);
+}
+
+.proposal_btn:hover,
+.proposal_btn:focus {
+  background-color: var(--dark-green-color);
+}
+
+.proposal__img {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .container__proposal {
+    padding: 64px 18px 88px 18px;
+  }
+
+  .proposal__content-wrapper::before {
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    transform: rotate(-9.86deg);
+  }
+
+  .proposal__content-wrapper::after {
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    transform: rotate(10.65deg);
+  }
+
+  .proposal__content {
+    padding: 64px 0 64px 32px;
+  }
+
+  .proposal__header {
+    margin-bottom: 88px;
+    padding-left: 14px;
+    width: 640px;
+  }
+
+  .proposal__feature-list {
+    width: 451px;
+  }
+
+  .proposal__feature {
+    gap: 16px;
+  }
+
+  .proposal__img {
+    display: block;
+    position: absolute;
+    bottom: 64px;
+    right: -50px;
+  }
+
+  .proposal_btn {
+    width: unset;
+    padding: 16px 64px;
+    font-size: 18px;
+    line-height: 1.55;
+  }
+}
+
+@media only screen and (min-width: 1280px) {
+  .container__proposal {
+    padding: 64px 64px 144px 64px;
+  }
+
+  .proposal__content-wrapper::before {
+    top: calc((var(--proposal-front-card-height) - var(--proposal-mid-card-height)) / 2);
+    left: calc((var(--proposal-front-card-width) - var(--proposal-mid-card-width)) / 2);
+    width: 1050px;
```

Comment:
don't need fixed width, height

Reply:
According to template, we have different dimensions (width, height) for middle and back cards in desktop layout